src/index.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Pages from "./components/Pages";
+import { MeatInfoProvider } from "./providers/MeatInfoProvider";
+import { ModalsConditionalsProvider } from "./providers/ModalsConditionalsProvider";
+import { CartInfoProvider } from "./providers/CartInfoProvider";
+
+
+
+ReactDOM.render(
+  <React.StrictMode>
+    <ModalsConditionalsProvider>
```

Comment:
Please separate this into an App component.
You should not putt all this stuff into the render function.